src/Controller/Component/AuthorizationComponent.php

```diff
@@ -86,4 +88,30 @@ protected function getIdentity(ServerRequestInterface $request)
 
         return $identity;
     }
+
+    /**
+     * Model authorization handler.
+     *
+     * @return void
+     */
+    public function authorizeModel()
+    {
+        $this->authorize($this->getController()->loadModel());
```

Comment:
For a view/edit/delete operation does is it useful to check if the user has access in general? Personally I would like to know if the current user is allowed to modify/access the specific records they are working on.

Reply:
The access for individual entities should be checked using entity policies IMO.

So `$this->Authorization->authorize($entity)` in action body. I’m not a fan of loading entities magically and checking access. Feels like a job for a plugin, Crud for instance.

Reply:
I can add a config option for supported actions if that’s your concern.

Reply:
Yeah I don't want to magically load entities either. I could see this being useful for add at least.